plugins/catalog-import/src/components/EntityListComponent/EntityListComponent.tsx

```diff
@@ -87,6 +85,17 @@ export const EntityListComponent = (props: EntityListComponentProps) => {
     );
   };
 
+  function sortEntities(entities: Array<CompoundEntityRef | Entity>) {
+    return entities.sort((a, b) =>
+      entityPresentationApi
+        .forEntity(stringifyEntityRef(a))
+        .snapshot.entityRef.localeCompare(
```

Comment:
hm ok ideally we would wait for https://github.com/backstage/backstage/pull/24595 and make this promise based so we can get an actual value (the snapshot is just a preliminary), but in this context I think let's go with what you have here - it's good enough